mesos-marathon/centos/README.md

```diff
@@ -1,34 +1,138 @@
 ---
 layout: guides
-title: "Using Weave with Mesos Marathon on CentOS"
+title: "Using Weave with Apache Mesos & Marathon on CentOS"
 permalink: /guides/platform/mesos-marathon/os/centos/cloud/vagrant/index.html
 tags: docker, mesos, marathon, cli, vagrant, virtualbox, dns, ipam
 ---
 
-In this guide I will demonstrate how to get Weave working with Mesos Marathon on CentOS 7 using Vagrant.
+##What You Will Build## 
```

Comment:
The redering on github makes leading `##` appear as part of the title here for some reasomn... In any case, you don't need to have it, just the preceding one is enough ;)